misc/python/materialize/git.py

```diff
@@ -83,3 +84,12 @@ def get_version_tags(*, fetch: bool = True) -> List[semver.VersionInfo]:
             print(f"WARN: {e}", file=sys.stderr)
 
     return sorted(tags, reverse=True)
+
+
+def is_ancestor(earlier: str, later: str) -> bool:
+    """True if earlier is in an ancestor of later"""
+    try:
+        spawn.capture(["git", "merge-base", "--is-ancestor", earlier, later])
+    except subprocess.CalledProcessError:
```

Comment:
`spawn` reexports this so I think you can just do `spawn.CalledProcessError`?

Reply:
Ah I thought of that as not being an intentional reexport, usually in python I only import from where exceptions are defined, or modules that opt-in to re-exports by adding the imported item to the `__all__` list.

So I could, but I'd rather explicitly opt-in, either by defining `__all__` in spawn (which... is always a little fragile/annoying) or by doing something like `materialize.errors.CalledProcessError = CalledProcessError` or making it otherwise explicit. I suppose that using mypy means that this is less of any issue than it used to be, so 🤷‍♂️ I could just do it.

Reply:
Feel free to just delete the reexport and make everything use subprocess directly since it sounds like I’m violating Python convention here!